mustache_test.go

```diff
@@ -107,6 +124,26 @@ func TestObjectOutput(t *testing.T) {
 	}
 }
 
+func TestObjectOutputJsonEscaped(t *testing.T) {
+	inputTemplate := strings.NewReader("Raw output here: {{.}}")
+	//inputData := map[string]map[string]string{"foo": {"bår": "baz"}}
+	inputData := map[string]map[string]string{"foo": {"bar": "baz"}}
+	template := New(JsonEscape())
+	err := template.Parse(inputTemplate)
+	if err != nil {
+		t.Error(err)
```

Comment:
nit: maybe fatal? similar for other places where you think not expected to have errors.